scripts/populate_panels/load_g2p_data.pl

```diff
@@ -1068,3 +1075,107 @@ sub add_public_comments {
   return $count;
 
 }
+
+sub add_ontology_accession {
+  my ($disease, $disease_mim, $disease_mondo) = @_;
+
+  if (defined($disease_mim) && !defined($disease_mondo)){
+    my @mondo = get_ontology_accession($disease_mim, $disease_mondo);
+    if (scalar @mondo > 0){
+      my $attribute = "Data source";
```

Comment:
These MONDO terms are being assigned by OLS, rather than the data supplier, so 'Data source' is not appropriate